### DISCUSSION TITLE
[Office Hours] Pulling Step Count data

### DISCUSSION THREAD

**Discussion by jennxu23** [CS342/.github#30] [Office Hours] [answered]
### Related Discussion

I'm trying to pull the step count data from Cardinal Kit. 

### Reproduction

N/A

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
Solved this in the office hours today by pointing @jennxu23 to the [`feature/charts`](https://github.com/StanfordBDHG/CardinalKitTemplateApplication/tree/feature/charts) in the [CardinalKitTemplateApplication](https://github.com/StanfordBDHG/CardinalKitTemplateApplication).
